docs/user-documentation/users.md

```diff
@@ -0,0 +1,36 @@
+# Managing Users
```

Comment:
Providing link to this documentation https://www.drupal.org/docs/user_guide/en/user-new-user.html would be useful.  Perhaps like:

Further Reading:
* [Drupal.org Managing User Acccounts](https://www.drupal.org/docs/user_guide/en/user-chapter.html)